bases/node-ts.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "display": "Node with TypeScript (TS >=5.8 ONLY)",
+    "docs": "https://devblogs.microsoft.com/typescript/announcing-typescript-5-8-beta/#the---erasablesyntaxonly-option",
+    "_version": "23.6.0",
+    "compilerOptions": {
+        "lib": [
```

Comment:
I don't think we should set any of the lib/target, and just have people use this config in an extends array with another config.

Reply:
That makes sense, let me rewrite it